protocols/directdefund/directdefund.go

```diff
@@ -56,17 +56,25 @@ func isInConsensusOrFinalState(c *channel.Channel) (bool, error) {
 // GetChannelByIdFunction specifies a function that can be used to retreive channels from a store.
 type GetChannelByIdFunction func(id types.Destination) (channel *channel.Channel, ok bool)
 
+// todo: #420 assume name and godoc from GetTwoPartyLedgerFunction
```

Comment:
this should be done now, #420 is closed already

Reply:
Done in https://github.com/statechannels/go-nitro/pull/602/commits/19b0cf43f2778592a0b223b7e48732876d9d6c25